pubsub/api/Pubsub.Samples/UpdatePushConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_update_push_configuration]
+
+using Google.Cloud.PubSub.V1;
+
+public class UpdatePushConfigurationSample
+{
+    public void UpdatePushConfiguration(string projectId, string subscriptionId, string pushEndpoint)
+    {
+        SubscriberServiceApiClient subscriber = SubscriberServiceApiClient.Create();
+        SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId, subscriptionId);
+
+        PushConfig pushConfig = new PushConfig { PushEndpoint = pushEndpoint };
+
+        subscriber.ModifyPushConfig(subscriptionName, pushConfig);
```

Comment:
If this returns the modified subscription, return it here so in the test we don't have to list all the subscriptions.

Reply:
it doesn't return updated subscription.

Reply:
OK